docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-installation.md

```diff
@@ -34,7 +34,7 @@ If you have the disk (say /dev/sdb), then you can use the below command to creat
 $ zpool create zfspv-pool /dev/sdb
 ```
 
-You can also create mirror or raidz pool as per your need. See [here](https://github.com/openzfs/zfs) for more information.
+You can also create mirror or raidz pool as per your need. Refer [Local PV ZFS](https://github.com/openzfs/zfs) for more information.
```

Comment:
This needs to point to openzfs docs and not their github repository. The hyperlink also needs to change.

[openzfs docs](https://openzfs.github.io/openzfs-docs/)

Reply:
> This needs to point to openzfs docs and not their github repository. The hyperlink also needs to change.
> 
> [openzfs docs](https://openzfs.github.io/openzfs-docs/)

This comment is yet to be addressed.

Reply:
This is already addressed. :)
<img width="665" alt="image" src="https://github.com/user-attachments/assets/ba90fbbe-3cfd-448f-9b54-93a4b1055317">

Reply:
Oh.. missed it !!